.github/workflows/python.yaml

```diff
@@ -183,6 +184,10 @@ jobs:
           # test if compatible with older versions
           pip install pandas==1.5.3 pyarrow\<12.0.0 sqlalchemy\<2
         fi
+        if [[ "$MODULE" == "jax" ]]; then
+          # test jax
+          pip install jax>=0.4.0 jaxlib>=0.4.0
```

Comment:
I found they are not installed, try to add quotes like this, `pip install "jax>=0.4.0" "jaxlib>=0.4.0"`.